dev/source/docs/building-setup-linux.rst

```diff
@@ -24,6 +24,9 @@ Clone ArduPilot repository
 .. include:: git-clone.rst
     :start-after: inclusion-marker-do-not-remove
     :end-before: Cloning with the GitHub GUI (Windows or MAC)
+    
+.. note::in case you get a connection error because of firewalls type command :
```

Comment:
You could say:
"Some firewalls do not allow ssh access which can cause submodule updates to fail, in this instance you can tell git to unilaterally use https through the following command:"

Reply:
ook , thanks!